examples/synthetic_data/syn_text_pii_scanner_example.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<p>Can we please add a description of what the example will be about? Something like:</p><p> </p><p><strong>Goal of the example</strong></p><p> </p><p><strong>Steps that will be showcased:</strong></p><ol><li>Load the TAB dataset and train a NERLongformer</li><li>Load a synthesized dataset based on X</li><li>Assess the presense of personal identifiable information (PII) that overlaps within the original and the synthetic documents</li><li>Showcase how public PIIs (PIIs present in many documents) can be excluded to focus only on the more sensitive non-public PIIs.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/LeakPro/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>